Directory.Build.props

```diff
@@ -0,0 +1,66 @@
+<Project>
+
+  <PropertyGroup>
+    <Authors>Microsoft</Authors>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <PackageIconUrl>https://raw.githubusercontent.com/Microsoft/UWPCommunityToolkit/master/build/nuget.png</PackageIconUrl>
+    <PackageProjectUrl>https://github.com/Microsoft/UWPCommunityToolkit</PackageProjectUrl>
+    <PackageLicenseUrl>https://github.com/Microsoft/UWPCommunityToolkit/blob/master/license.md</PackageLicenseUrl>
+    <PackageReleaseNotes>v2.0: https://github.com/Microsoft/UWPCommunityToolkit/releases/tag/v2.0.0 </PackageReleaseNotes>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+
+
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)Toolkit.ruleset</CodeAnalysisRuleSet>
+    
+    <DefaultLanguage>en-US</DefaultLanguage>
+
+    <IsTestProject>$(MSBuildProjectName.Contains('Test'))</IsTestProject>
+    <IsUwpProject>$(MSBuildProjectName.Contains('Uwp'))</IsUwpProject>
+    <IsSampleProject>$(MSBuildProjectName.Contains('Sample'))</IsSampleProject>
+
+    <UwpMetaPackageVersion>5.3.4</UwpMetaPackageVersion>
+    <DefaultTargetPlatformVersion>15063</DefaultTargetPlatformVersion>
+    <DefaultTargetPlatformMinVersion>14393</DefaultTargetPlatformMinVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsUwpProject)' == 'true'">
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(IsTestProject)' != 'true' and '$(SourceLinkEnabled)' != 'false'">
+    <PackageReference Include="SourceLink.Create.CommandLine" Version="2.2.0" PrivateAssets="All" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(IsTestProject)' != 'true' and '$(IsSampleProject)' != 'true'">
+    <!--<PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="15.3.83" PrivateAssets="all" />-->
+    <PackageReference Include="StyleCop.Analyzers" Version="1.0.2" PrivateAssets="all" />
+
+    <EmbeddedResource Include="**\*.rd.xml" />
+    <Page Include="**\*.xaml" Exclude="**\bin\**\*.xaml;**\obj\**\*.xaml" SubType="Designer" Generator="MSBuild:Compile" />
+    <Compile Update="**\*.xaml.cs" DependentUpon="%(Filename)" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsUwpProject)' == 'true'">
+    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.0.9" PrivateAssets="all" />
+  </ItemGroup>
+
+  
+  <PropertyGroup>
+    <NerdbankGitVersioningVersion>2.0.37-beta</NerdbankGitVersioningVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Nerdbank.GitVersioning" Version="2.0.37-beta" PrivateAssets="all" />
```

Comment:
Shouldn't we use the `NerdbankGitVersioningVersion` property above for the `Version` parameter here?

Reply:
You'd think so, but it breaks in some cases currently. I don't have the issue handy, but the problem is that there was a case where the unevaluated value was passed to NuGet and NuGet just saw `$(NerdbankGitVersioningVersion)` and not the value.

Reply:
Weird one... fair enough then!

Reply:
Here's the issue link https://github.com/dotnet/project-system/issues/2129